docs/advanced/customization-guide.md

```diff
@@ -74,6 +74,57 @@ Now, on X86 platforms the feature label appears after doing `modprobe dummy` on
 a system and correspondingly the label is removed after `rmmod dummy`. Note a
 re-labeling delay up to the sleep-interval of nfd-worker (1 minute by default).
 
+### Taints
+
+In some circumstances it is desirable keep nodes with specialized hardware away from
+running general workload and instead leave them for pods that need the specialized
+hardware. One way to achieve it is to taint the nodes with the specialized hardware
+and add corresponding toleration to pods that require the special hardware. NFD
+offers node tainting functionality which is disabled by default. User can define
+one or more custom taints via the `taints` field of the NodeFeatureRule CR. The
+same rule-based mechanism is applied here and the NFD taints only rule matching nodes.
+
+*taints* is a list of taint entries and each entry can have `key`, `value` and `effect`,
+where the `value` is optional. Effect could be `NoSchedule`, `PreferNoSchedule`
+or `NoExecute`. To learn more about the meaning of these effects, check out k8s [documentation](https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration/).
+
+To enable the tainting feature, `--enable-taints` flag needs to be set to `true`.
+If the flag `--enable-taints` is set to `false` (i.e. disabled), taints defined in
+the NodeFeatureRule CR have no effect and will be ignored by the NFD master.
```

Comment:
I think we should document the `taints` field in a separate subsection under [Fields](https://kubernetes-sigs.github.io/node-feature-discovery/master/advanced/customization-guide.html#fields) like the other fields, too. I think this paragraph could go there. And we need to state there that the `taints` field is ONLY available for `NodeFeatureRule` objects, not for the `custom` rules of nfd-worker.

Reply:
Ah sorry I misunderstood you earlier. Fixed now. Thanks